.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Existing functionality does not work as intended
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe The Bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Set up ...
+2. Execute ...
+3. Confirm ...
+4. See error
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Used Versions:**
+ - `node -v`: [e.g. v11.12.0]
+ - `npm -v`: [e.g. 6.12.0]
+ - `sap-cloud-sdk -v`: [e.g. sap-cloud-sdk-cli/0.0.1 darwin-x64 node-v11.12.0]
```

Comment:
It might make sense in the future to add some thing like a `diagnose` command to the CLI so that users don't have to collect all the versions manually but can run something like `cloud-sdk-cli diagnose` and simply copy the output here.
That's for the future, though.

Reply:
I agree but maybe it's not necessary because the node version is already included in the `-v` command and the npm version might not be relevant. We should evaluate once we gathered some debugging experience.